$OpenBSD: patch-configure_in,v 1.1 2013/09/30 09:46:08 dcoppa Exp $

OpenBSD lacks 'nm -D'

setuid root/setgid utmp are handled by PLIST

--- configure.in.orig	Mon Aug  5 14:48:58 2013
+++ configure.in	Wed Sep 25 10:20:52 2013
@@ -307,7 +307,7 @@ AC_SUBST(IMAGELIB_LIBS)
 for ml_cv_lib_path in /lib /usr/lib /lib/* `echo $LIBRARY_PATH | tr ":" " "` ; do
 	libm_files=`ls $ml_cv_lib_path/libm\\.so* $ml_cv_lib_path/libm\\.sl* 2> /dev/null`
 	for libm_file in $libm_files ; do
-		if test -z "$ml_cv_libm_dir" -a -n "`$NM -D $libm_file | $GREP pow`" ; then
+		if test -z "$ml_cv_libm_dir" -a -n "`$OBJDUMP -T $libm_file | $GREP \ pow$`" ; then
 			ml_cv_libm_dir=`dirname $libm_file`
 		fi
 	done
@@ -609,19 +609,9 @@ AC_ARG_ENABLE(pty_helper,
 	pty_helper=$enable_pty_helper)
 if test "$utmp" = "yes" -a "$pty_helper" != "yes" ; then
 	UTMP_CFLAGS="-DUSE_UTMP"
-	has_utmp=`grep utmp /etc/group 2>/dev/null`
-	if test "$has_utmp" ; then
-		INSTALL_OPT="-m 2755 -g utmp"
-	else
-		INSTALL_OPT="-m 4755 -o root"
-	fi
-else
-	INSTALL_OPT="-m 755"
 fi
 
-if test "$gui" = "fb" ; then
-	INSTALL_OPT="-m 4755 -o root"
-fi
+INSTALL_OPT="-m 0555 -s"
 
 AC_SUBST(UTMP_CFLAGS)
 AC_SUBST(INSTALL_OPT)
